src/tables.rs

```diff
@@ -192,10 +192,19 @@ impl BlockSize {
     }
 }
 
-pub static dav1d_txfm_dimensions: [TxfmInfo; TxfmSize::COUNT] = {
+pub const fn dav1d_txfm_size<const TX: usize>() -> TxfmSize {
+    let Some(size) = TxfmSize::from_repr(TX) else {
+        unsafe {
+            std::hint::unreachable_unchecked();
```

Comment:
This can just be a panic if we always call this in `const` contexts, which we can.  So no unsafe needed.

Reply:
We won't be able to have a panic message referencing the invalid value or the number of `TxfmSize` items, but I've pushed a commit to panic here.

Reply:
Yeah that should be fine.  It'd just be a compiler error and it's very unlikely to happen accidentally.